bento_lib/package.cfg

```diff
@@ -1,5 +1,5 @@
 [package]
 name = bento_lib
-version = 5.4.0
+version = 5.4.1.8
```

Comment:
this is not a valid python version - if you are making dev releases, use the python version format https://peps.python.org/pep-0440/ (these would be `5.5.0a1` etc)

as it stands now i am going to delete these release from pypi because this versioning scheme indicates they are prod versions to other services which use bento_lib, which would screw up any updates done based on version.